src/app/shared/components/template/components/task-card/task-card.component.html

```diff
@@ -3,26 +3,29 @@
     [class]="'card-wrapper landscape ' + style"
     [attr.data-variant]="variant"
     (click)="triggerActions('click')"
+    [attr.data-status]="progressStatus"
   >
     <ng-container *ngIf="highlighted; else showProgress">
       <span class="badge highlighted-badge">
         {{ highlightedText }}
       </span>
     </ng-container>
     <ng-template #showProgress>
-      <span class="badge progress-badge">
-        <span class="circle" [class]="progressStatus"></span>
-        <img
-          *ngIf="progressStatus === 'inProgress'"
-          class="icon"
-          src="{{ inProgressIcon | plhAsset }}"
-        />
-        <img
-          *ngIf="progressStatus === 'completed'"
-          class="icon completed"
-          src="{{ completedIcon | plhAsset }}"
-        />
-      </span>
+      @if (inProgressIcon || completedIcon) {
+        <span class="badge progress-badge">
+          <span class="circle" [class]="progressStatus"></span>
+          <img
+            *ngIf="progressStatus === 'inProgress'"
```

Comment:
nit(non-blocking) It'd be nice to refactor these to use the `@if` Angular control flow syntax too